### PR TITLE
Portfolio: popup window

### PR DESCRIPTION
**Pull request by Yonashenok**
## what we did
* Making sure no linter errors
* Followed the Figma design
* added the features
   1. When user click on the button to check project details, the popup with details about the project appears.
   2.  When user click on close (X)  button, the popup disappears.
   3.  Added JavaScript array object to store the content information for all project
   4. I used JavaScript logic to display specific project information dynamically from an array object.

thanks😀

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,6 +83,92 @@ const scrollHandler = (event) => {
...
+    </p>
+
+    <p class="text-box hidden-m">
+    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Est ex ipsam
+    eligendi impedit, laudantium officia exercitationem non error ab
+    corrupti modi veniam, laborum eum earum vitae voluptates fugiat, sunt
+    aut. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quam
+    magnam facilis ex, est recusandae non fugiat odio ipsum ipsam iste ab
+    nostrum aperiam dolor? Modi doloremque beatae voluptatum maxime hic!  
+    </p>
```
Comment (DuaneDave) [inline index.js]: Good job using the data from the object filtered to render this modal, however, you aren't pulling out the description into this paragraph tag. Consider doing so in order to fully satisfy the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.)

File: index.js [hunk 2]
```diff
@@ -23,6 +83,92 @@ const scrollHandler = (event) => {
...
+    height="32"
+    width="32"
+    stroke="#67798E"
+    class="w-6 h-6 crossIcon"
+  >
+    <path
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      d="M6 18L18 6M6 6l12 12"
+    />
```
Comment (DuaneDave) [inline index.js:114]: A lot of work you've put in to make this work, good job 👍, However, you haven't paid so much attention to the modal is overflowing and it doesn't fit correctly on the browser on the desktop. On the mobile, the modal content fills the screen and the close button is hidden away. I recommend paying a close attention to the position of the modal to make it UI friendly and UX acceptable.

HINT: For the mobile, you can make the modal a scrollable one.


desktop

toonmuvj

mobile

mobile

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Yonashenok 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ You implementation for the modal is quite superb, good job 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Yonashenok, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter Errors :100: 
- Good PR Description :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (GedeonTS) [inline index.js]: - As Requested by your last reviewer [Here](https://github.com/Yonashenok/My-portfolio/pull/9/files#r1196303737), Kindly remove this paragraph as you have already rendered the description for each project using a JavaScript array.

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Yonashenok, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter Errors :100: 
- Good PR Description :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Yonashenok) [reply index.js]: # what I did
* remove the paragraph from JavaScript file

**Review by danmainah — APPROVED** [review]
Hi @Yonashenok , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Linter setup correctly and all checks are passing
✔️ Good design in your popup windows

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**